decidim-debates/spec/types/debate_type_spec.rb

```diff
@@ -97,6 +97,48 @@ module Debates
           expect(response).to include("reference" => model.reference.to_s)
         end
       end
+
+      context "when participatory space is private" do
```

Comment:
Can you add another example for `when participatory space is private and transparent` 🙏🏽 ?

Reply:
done in b46c0e3468